outlines/using-packages.md

```diff
@@ -0,0 +1,34 @@
+# Using packages
+
+1. Packages vs modules
```

Comment:
I just thought it might be useful for people to realise there's still a need to use packages even in a module world?